docs/1-project-setup.md

```diff
@@ -100,7 +100,7 @@ Ensure each of the following secrets have been set, using the values you entered
 
 ### Create the Azure Service Principal
 
-You need to configure an [Azure Service Principal](https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli) to allow the pipeline to login using your identity and to work with Azure resources on your behalf. You will save the access token for the service principal in the GitHub Secrets for your repository.
+You need to configure an [Azure Service Principal](https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli) to allow the workflow to login using your identity and to work with Azure resources on your behalf. You will save the access token for the service principal in the GitHub Secrets for your repository.
 
 A Powershell script [./setup/create_sp.ps1](./setup/create_sp.ps1) is provided in this repo to make this simple and the easiest way to run the script is to use [Azure Cloud Shell](https://shell.azure.com).
 
```

Comment:
Nit for line 109. Do we need to give them 3 ways to get to Azure Cloud Shell?

Reply:
Agreed. Reduced to the button and the Azure Portal.